assets/js/components/Dashboard/PlacesPage.tsx

```diff
@@ -68,6 +71,7 @@ const PlacesList: ComponentType<PlacesListProps> = ({
     activeEventKeys,
   } = usePlacesPageContext();
   const dispatch = usePlacesPageDispatchContext();
+  const prevPlaceIds = usePrevious(places)?.map((place) => place.id);
```

Comment:
Same here, you can nullish-coalesce this into an empty array if previous places is undefined.